com.unity.UnityHub.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: com.unity.UnityHub
+branch: stable
```

Comment:
`branch` is implied stable on Flathub, you can remove this, then it will also use a `master` branch when built elsewhere, which is useful, because it prevents conflicts.